NCG613a_Project1.ipynb

```diff
@@ -639,6 +639,646 @@
+       "      <td>City of London Police</td>\n",
+       "      <td>-0.097601</td>\n",
+       "      <td>51.520699</td>\n",
+       "      <td>On or near Carthusian Street</td>\n",
+       "      <td>E01000001</td>\n",
+       "      <td>City of London 001A</td>\n",
+       "      <td>Shoplifting</td>\n",
+       "      <td>Status update unavailable</td>\n",
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            Crime ID    Month  \\\n",
+       "0                                                NaN  2021-01   \n",
+       "1                                                NaN  2021-01   \n",
+       "2  455f0a5e2aa0b0998be8adce21c42e362d85cb5cd0e31a...  2021-01   \n",
+       "3  19f06058a3b0aa0e79eabc017b48862f2a6a06645b2f5c...  2021-01   \n",
+       "4  c1554ced3041294d44478a0a789b2a5a52bff2ad27674b...  2021-01   \n",
+       "\n",
+       "             Reported by           Falls within  Longitude   Latitude  \\\n",
+       "0  City of London Police  City of London Police  -0.097601  51.520699   \n",
+       "1  City of London Police  City of London Police  -0.098572  51.516767   \n",
+       "2  City of London Police  City of London Police  -0.097334  51.521567   \n",
+       "3  City of London Police  City of London Police  -0.098572  51.516767   \n",
+       "4  City of London Police  City of London Police  -0.097601  51.520699   \n",
+       "\n",
+       "                        Location  LSOA code            LSOA name  \\\n",
+       "0   On or near Carthusian Street  E01000001  City of London 001A   \n",
+       "1  On or near King Edward Street  E01000001  City of London 001A   \n",
+       "2         On or near Fann Street  E01000001  City of London 001A   \n",
+       "3  On or near King Edward Street  E01000001  City of London 001A   \n",
+       "4   On or near Carthusian Street  E01000001  City of London 001A   \n",
+       "\n",
+       "              Crime type                          Last outcome category  \\\n",
+       "0  Anti-social behaviour                                            NaN   \n",
+       "1  Anti-social behaviour                                            NaN   \n",
+       "2            Other theft  Investigation complete; no suspect identified   \n",
+       "3            Other theft  Investigation complete; no suspect identified   \n",
+       "4            Shoplifting                      Status update unavailable   \n",
+       "\n",
+       "   Context  \n",
+       "0      NaN  \n",
+       "1      NaN  \n",
+       "2      NaN  \n",
+       "3      NaN  \n",
+       "4      NaN  "
+      ]
+     },
+     "execution_count": 96,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Read the crime data of 2021 for City of London\n",
+    "crime_col2021 = pd.read_csv('2021-city-of-london-street.csv')\n",
```

Comment:
Can you share this file on Teams. I assume its the combination of all the monthly crime data for City of London for 2021

Reply:
Uploaded.